src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius) {
+        setRadius(radius);
+        setColor();
+    }
```

Comment:
в конструкторі не варто викликати метод для того щоб присвоїти полю якесь значення
```suggestion
    public Circle(int radius, Color color) {
        this.radius = radius;
        setColor(color);
    }
```

Reply:
done